src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,25 @@
+package duke;
+
+import duke.command.AddCommand;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ParserTest {
+    
+    @Test
+    public void parse_todoBorrowBook_newAddCommand() throws DukeException {
```

Comment:
Great job naming test methods using underscores and following the three part format!

Reply:
Thank you for the review! :)